### PR TITLE
better integration test file path changes detection

### PR DESCRIPTION
**Pull request by neuronull**

### CODE REVIEW

File: src/sources/http_client/client.rs [hunk 1]
```diff
@@ -116,7 +116,7 @@ const fn default_http_method() -> HttpMethod {
 fn query_examples() -> HashMap<String, Vec<String>> {
     HashMap::<_, _>::from_iter(
         [
-            ("field".to_owned(), vec!["value".to_owned()]),
```
Comment (spencergilbert) [inline src/sources/http_client/client.rs:119]: Was this just a test to trigger changes? Should it be kept in?

**Review by spencergilbert — APPROVED** [review]

File: src/sources/http_client/client.rs (hunk 1 above)
Comment (neuronull) [reply src/sources/http_client/client.rs:119]: 👋 this is a pr in the plaground repo :)

Comment (spencergilbert) [reply src/sources/http_client/client.rs:119]: 😂 Didn't even notice

Comment (neuronull) [reply src/sources/http_client/client.rs:119]: the good news is that im going to make a real PR for it shortly

**Pull request closed without merging**